05week/bankAccount.js

```diff
@@ -0,0 +1,139 @@
+//For this assignment we are creating 2 classes that can be used to represent a Bank Account and the transactions it holds.
+
+
+/*BankAccount Class - This class represents a Bank Account
+-----------------
+1. The class should have the following fields:
+	a. accountNumber - String representing the account number 
+	b. owner - String representing the owner of the account
+	c. transactions - an array of transactions representing the history of all transactions associated with this account
+*/
+/*
+2. The constructor should take in the following input:
+	a. accountNumber - the account number
+	b. owner - this is the name of the person who owns the account
+	c. NOTE: that the when an account is created, you should initialize the transactions array to be an empty array. 
+*/
+
+class BankAccount {
+  constructor(accountNumber, owner){
+    this.accountNumber = accountNumber;
+    this.owner = owner;
+		this.transactions = [];
+		this.accountBalance = 0; 
+		
+    //NOTE: that the when an account is created, you should initialize the transactions array to be an empty array.  
+  }
+    balance(){
```

Comment:
`balance()` should loop through the transactions to get the balance, instead of being stored in the bankaccount.